relay-server/src/actors/upstream.rs

```diff
@@ -139,7 +139,7 @@ impl UpstreamRequestError {
     fn is_network_error(&self) -> bool {
         match self {
             Self::SendFailed(_) => true,
-            Self::ResponseError(code, _) => matches!(code.as_u16(), 502 | 503 | 504),
+            Self::ResponseError(code, _) => matches!(code.as_u16(), 502..=504),
```

Comment:
I believe this is actually harder to read than the previous diff state. Do you think the new version is more readable?

Reply:
That's what `lint` wants. For me personally it does not matter, both are equally ok.